File: app/src/main/java/com/mrcaracal/mobilgezirehberim/SplashScreen.java
package com.mrcaracal.mobilgezirehberim;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;
import android.view.animation.Animation;
import android.view.animation.AnimationUtils;
import android.widget.TextView;

public class SplashScreen extends AppCompatActivity {

    TextView txt_giris_yazisi;
    Animation animation;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splash_screen);

        txt_giris_yazisi = findViewById(R.id.txt_giris_yazisi);

        animation = AnimationUtils.loadAnimation(getApplicationContext(), R.anim.anim1);
        txt_giris_yazisi.startAnimation(animation);

        new Handler().postDelayed(new Runnable() {
            @Override
            public void run() {
                Intent intent = new Intent(SplashScreen.this, Login.class);
                startActivity(intent);
            }
        }, 3000);

    }
}

File: app/src/main/java/com/mrcaracal/mobilgezirehberim/SplashScreen.kt
package com.mrcaracal.mobilgezirehberim

import android.content.Intent
import android.os.Bundle
import android.os.Handler
import android.os.Looper
import android.view.animation.Animation
import android.view.animation.AnimationUtils
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity
import com.mrcaracal.mobilgezirehberim.login.Login

class SplashScreen : AppCompatActivity() {

    private lateinit var txt_giris_yazisi: TextView
    private var animation: Animation? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash_screen)
        txt_giris_yazisi = findViewById(R.id.txt_giris_yazisi)
        animation = AnimationUtils.loadAnimation(applicationContext, R.anim.anim1)
        txt_giris_yazisi.startAnimation(animation)

        Handler(Looper.myLooper() ?: return).postDelayed({
            val intent = Intent(this@SplashScreen, Login::class.java)
            startActivity(intent)
        }, 2000)

    }

}